data/api/server-server/definitions/pdu_v11.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+type: object
+title: Persistent Data Unit
+description: A persistent data unit (event) for room version 4 and beyond.
+example:
+  $ref: "../examples/pdu_v11.json"
+allOf:
+  # v11 is the v4 event, but without redacts. Copy the auth_events/prev_events
+  # from pdu_v4.yaml and hashes and signatures from pdu_v3.yaml.
```

Comment:
I debated making a `pdu_v3_hash_sigs.yaml` and `pdu_v4_auth_prev_events.yaml` and using each twice, but that seemed like overkill? WDYT?